layouts/partials/taxonomy/categories.html

```diff
@@ -4,6 +4,6 @@
     {{- if gt $index 0 }}
       <span class="separator">•</span>
     {{- end }}
-    <a href="{{ ( print "categories/" ( . | urlize ) ) | relLangURL }}">{{ . }}</a>
+    <a href="{{ ( print "categories/" ( . | urlize ) "/" ) | relLangURL }}">{{ . }}</a>
```

Comment:
```suggestion
    <a href="{{ ( printf "categories/%s/" ( . | urlize ) ) | relLangURL }}">{{ . }}</a>
```
It is better this way.